contact/index.html

```diff
@@ -29,16 +29,42 @@ <h1 class="header-left">
         <div class="contact-container">
             <div class="contact-title">
                 <h2>ご相談・お問い合わせ</h2>
-                    <p class="contact-notes">
-                        必須内容をご記入の上、お問い合わせください。内容確認が終了次第、3営業日以内にご返信差し上げます。<br>
-                        お問い合わせ内容によってはお電話にて回答させていただく場合もございますので、あらかじめご了承ください。
-                    </p>
+                <p class="contact-notes">
+                    必須内容をご記入の上、お問い合わせください。内容確認が終了次第、3営業日以内にご返信差し上げます。<br>
+                    お問い合わせ内容によってはお電話にて回答させていただく場合もございますので、あらかじめご了承ください。
+                </p>
+            </div>
 
-        <!-- 問合せフォーム内容 --> 
-            <form action="program.php" method="post">
-                <dl>
-                </dl>
-            </form>
+            <!-- 問合せフォーム内容 --> 
+            <div class="contact-form">
+                <form action="program.php" method="post">
+                    <div class="form-rows"></div>
```

Comment:
`form-rows` が閉じてしまっているので、この行の閉じタグを削除して、複数の`form-row`をラップできるよう適切な位置に閉じタグを追加ください🙏
```suggestion
                    <div class="form-rows">
```

Reply:
すません💦</div>位置調整しました！